src/index.js

```diff
@@ -69,6 +69,9 @@ export default class VueI18n {
     const messages: LocaleMessages = options.messages || {}
     const dateTimeFormats = options.dateTimeFormats || {}
     const numberFormats = options.numberFormats || {}
+    const icon = options.icons || false
```

Comment:
I think the name `icons` is too general to deliver an intent of the option.

And the existing `MissingHandler` extension point might be ussable to implement this feature
How about add factories for MissingHandler, like this?

```js
function createPrefixedMissingHandler(prefix) {
  const prefix_  = typeof prefix === "string" ? prefix : "🔥";
  return (locale, key) => {
    return `${prefix_} ${key}`;
  };   
}

VueI18n.missingHandlers = {
  createPrefixedMissingHandler,
  // add more predefined handler/handler factory in future
}
```

This also allow users to customise prefix, e.g show `(untranslated)` instead of fancy icon.

Reply:
Changes made